.test/config.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+globalTests+=(
```

Comment:
https://github.com/docker-library/official-images/blob/master/test/README.md#alternate-config-files suggests this would no longer run the official images tests, no?

Reply:
I think you are right, I need to move local tests away from `globalTests`.

Reply:
I've updated the configuration, should probably work better now.